test/end-to-end.js

```diff
@@ -19,6 +19,8 @@ var TestDataBuilder = require('./helpers/test-data-builder');
 var ref = TestDataBuilder.ref;
 var given = require('./helpers/given');
 var should = require('chai').should();
+var loopback = require('loopback');
+var boot = require('loopback-boot');
```

Comment:
Hmm, I think this may not work when the scaffolded application uses a different loopback or loopback-boot version than we use in loopback-workspace. Considering that fix is not straightforward and the situation may actually never happen, I think it's ok to leave these two lines as they are, but please add a comment pointing out this possible problem.